website/docs/d/database_pitr.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_database_pitr"
```

Comment:
page title should be the name of datasource "ibm_database_point_in_time_recovery"